src/PerimeterxHttpClient.php

```diff
@@ -35,9 +35,8 @@ public function __construct($config)
      * @inheritdoc
      * @return string
      */
-    public function send($url, $method, $json, $headers, $timeout = 0, $connect_timeout = 0)
+    public function send($pxCtx, $url, $method, $json, $headers, $timeout = 0, $connect_timeout = 0)
```

Comment:
not so important, but I would put pxCtx later in the function parameters (at least after the important parameters such as url and method)

Reply:
Done